xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/Main/SolrSearchMacros.xml

```diff
@@ -824,6 +824,11 @@ $value##
 
     #end
     #displaySearchForm()
+    #foreach ($extension in $services.uix.getExtensions("org.xwiki.platform.search.widget"))
```

Comment:
ok, I know this is a probably a pain for you but this needs to be discussed and agreed on the list since it's a new API and we cannot remove it thereafter. We need to decide:
* the scope (SOLR only, all search UIs)
* the name and the concept of "search widgets"
* whether it's one UIXP or 2. In the jira issue you describe 2 needs which are very different to me. You may be able to cover them with one UIXP with the current SOLR UI but an UIXP must work for any UI.

I'll comment in the jira issue. Thanks